include/nudb/error.hpp

```diff
@@ -30,6 +30,22 @@ enum class error
     */
     success = 0,
 
+    /** The specified key was not found.
+
+        Returned when @ref basic_store::fetch does not
+        find the specified key.
```

Comment:
The Returned comment does not add value and may mislead to think its the only ec value set with fetch. I would have the user expect any ec code to be set.